crates/sparrow-plan-execution/src/tests.rs

```diff
@@ -1,9 +1,9 @@
 use arrow_array::cast::AsArray;
 use arrow_array::{Int64Array, RecordBatch, TimestampNanosecondArray, UInt64Array};
 use sparrow_interfaces::source::{Source, SourceExt};
+use sparrow_io::in_memory::in_memory_source::InMemoryBatches;
```

Comment:
It seems like the `in_memory` is useful, but I'm not sure about exposing the `in_memory_source`. I'd suggest either `sparrow_io::in_memory::InMemoryBatches` or `sparrow_io::in_memory::source::InMemoryBatches`.